bot/exts/evergreen/emoji_count.py

```diff
@@ -17,73 +18,67 @@ class EmojiCount(commands.Cog):
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    def embed_builder(self, emoji: dict) -> discord.Embed:
+    def embed_builder(self, emoji: dict) -> [discord.Embed, str]:
         """Generates an embed with the emoji names and count."""
         embed = discord.Embed(
             color=Colours.orange,
             title="Emoji Count",
             timestamp=datetime.datetime.utcnow()
         )
+        msg = []
 
         if len(emoji) == 1:
             for key, value in emoji.items():
-                embed.description = f"There are **{len(value)}** emojis in the **{key}** category"
+                msg.append(f"There are **{len(value)}** emojis in the **{key}** category")
```

Comment:
Use `is` instead of `are` because there is only 1 emoji
```suggestion
                msg.append(f"There is **{len(value)}** emoji in the **{key}** category")
```

Reply:
actually, you have to check if len(value) is equal to 1, only then use `is`.